apps/launchpad/package.json

```diff
@@ -31,6 +32,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "recharts": "^2.15.0",
+    "remix-utils": "^7.6.0",
```

Comment:
Am I missing where this is used? I know we also used it in Portal

Reply:
I had to wrap the modal in ClientOnly otherwise it wasn't working properly. That comes from the remix-utils package.